app/views/comments/_comment_form.html.erb

```diff
@@ -0,0 +1,6 @@
+ <%= form_with model:[@post, @post.comments.build], method: :post, local: true do |form| %>
```

Comment:
Please add a new view for comment form in `views/comments/new.html.erb` file which accepts input for creating comments.

Reply:
resolved